puppet/spec/spec_helper.rb

```diff
@@ -10,9 +10,17 @@
 require 'tmpdir'
 require 'fileutils'
 require 'puppet'
+require 'puppet/util/puppetdb'
 require 'puppet/util/log'
-require 'puppet/util/puppetdb/command'
 
+def create_environmentdir(environment)
+  if not Puppet::Util::Puppetdb.puppet3compat?
+    envdir = File.join(Puppet[:environmentpath], environment)
+    if not Dir.exists?(envdir)
```

Comment:
Should we be using exist? instead of exists? here? It looks like exists? is deprecated